### DISCUSSION TITLE
500 Internal server error - After selecting translation.

### DISCUSSION THREAD

**Discussion by LaurensMapix** [kimai/kimai#3619] [General]
### Describe the issue

When I switch langueges in the settings to "nederlands" (Dutch), I get a 500 Internal server error:
I also get this error page when I choose doctor.

```
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.
```

This is on a newly installed Kimai2 instance.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kevinpapst/kimai2/discussions).

### Kimai version

1.28.1

### Which PHP version are you using?

7.4

### How do you run Kimai?

Self-Hosted

### Steps to reproduce the behavior

1. Go to 'User Preferences'
2. Select a different language.
3. See error

### Logfile

Nothing in the prod.log file.

### Additional context

Server: Linux xxx 5.4.0-131-generic #147-Ubuntu SMP Fri Oct 14 17:07:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
Browser: Chrome 106

**Comment by kevinpapst** [comment]
Please try the official demo before posting such "bugs".
I'll converted this to a discussion, because this is for sure not a Kimai bug.
Run `bin/console kimai:reload` and check if there are any errors shown.

**Reply by LaurensMapix** [reply]
I've tried a reload and I don't see any issues.
Now after I changed the languages, I'm no longer able to access anything, as all requests return a 500 error.
I could before the reload access the "/en" pages to reconfigure the language, but now even that won't work.

**Reply by kevinpapst** [reply]
Fix your file permissions, check var/logs/prod.log
